openwisp_firmware_upgrader/api/serializers.py

```diff
@@ -0,0 +1,24 @@
+from rest_framework import serializers
+from swapper import load_model
+
+Build = load_model('firmware_upgrader', 'Build')
+Category = load_model('firmware_upgrader', 'Category')
+UpgradeOperation = load_model('firmware_upgrader', 'UpgradeOperation')
+
+
+class BuildSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Build
+        fields = '__all__'
+
+
+class CategorySerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Category
+        fields = '__all__'
+
+
+class UpgradeOperationSerializer(serializers.ModelSerializer):
```

Comment:
shouldn't this be `BatchUpgradeOperation`? The mass upgrade operation should contain the info about the related upgrade operations in its detail view.